build/helm.bzl

```diff
@@ -102,7 +102,7 @@ def helm_tmpl(
     cmds = []
     set_args = []
     for k, v in values.items():
-        set_args = set_args + ["--set=%s=%s" % (k, v)]
+        set_args = set_args + ["--set=\"%s=%s\"" % (k, v)]
```

Comment:
Why is this necessary in this PR? I can only imagine it being `--set=creator=static`? If it is important, then please add a comment explaining why these MUST be quoted.

Reply:
Not necessary for this PR; but I think this could help prevent errors.